draft/spec/index.html

```diff
@@ -953,6 +965,127 @@ <h2>Versioned OCFL Object</h2>
 
     <section id="example-bagit-in-ocfl">
         <h2>BagIt in an OCFL Object</h2>
+        <p>
+            [[!BagIt]] is a popular file packaging specification, but unlike OCFL it does not provide a mechanism for
```

Comment:
I would advocate for less subjective language such as "popular". Maybe replace "popular" with "common"?

Reply:
Awesome? Tubular? Totally Rad? 

OK.